platform/pwl.c

```diff
@@ -1168,22 +1267,13 @@ registry_global (void               *data,
                                            &wl_shell_interface,
                                            version);
     } else if (strcmp (interface, wl_seat_interface.name) == 0) {
-        if (display->seat) {
-            g_warning ("%s: Multiple seats are not supported, ignoring "
-                       "wl_seat#%" PRIu32 ".", G_STRFUNC, name); 
-            interface_used = FALSE;
-        } else {
-            display->seat_version = MIN (version, WL_POINTER_FRAME_SINCE_VERSION);
-            display->seat = wl_registry_bind (registry,
-                                              name,
-                                              &wl_seat_interface,
-                                              display->seat_version);
-            static const struct wl_seat_listener seat_listener = {
-                .capabilities = display_on_seat_capabilities,
-                .name = display_on_seat_name,
-            };
-            wl_seat_add_listener (display->seat, &seat_listener, display);
-        }
+        uint32_t seat_version = MIN (version, WL_POINTER_FRAME_SINCE_VERSION);
+        struct wl_seat *wl_seat = wl_registry_bind (registry,
+                                                   name,
+                                                   &wl_seat_interface,
+                                                   seat_version);
```

Comment:
The indentation here is missing one space.

Reply:
done